docs/en/transform-v2/jsonpath.md

```diff
@@ -51,6 +60,14 @@ Support SeatunnelDateType
 
 > Jsonpath
 
+#### column_error_handle_way [Enum]
```

Comment:
Add a `column_error_handle_way` config in example? And please share more about priority between `column_error_handle_way` and `row_error_handle_way`.